src/107-Arduino-Servo-RP2040.cpp

```diff
@@ -66,6 +79,29 @@ void Servo::writeMicroseconds(uint16_t const pulse_width_us)
   pwm_set_chan_level(_slice_num, _channel, pulse_width_us);
 }
 
+void Servo::write(int const value)
+{
+  if (value < 0)
+    value = 0;
```

Comment:
I assume this line breaks now, since `value` is `const`, however this can be circumvented by adding a new variable and assigning to this. I think its worth it over the emphasis that the input parameter is a constant, unchangeable value.
i.e.
```C++
int angle = value;
value = max(0, value);
value = min(_max_angle, value);
```